packages/amplify-app/src/index.js

```diff
@@ -352,7 +353,12 @@ async function createJSHelperFiles() {
   console.log(`${emoji.get('white_check_mark')} Successfully added helper npm run scripts to your package.json.`);
 
   return new Promise((resolve, reject) => {
-    const npmInstall = spawn(npm, ['install', '--only=dev'], { cwd: process.cwd(), env: process.env, stdio: 'inherit' });
+    const npmInstall = spawn(npm, ['install', '--only=dev'], {
+      cwd: process.cwd(),
+      env: process.env,
+      stdio: 'inherit',
+      shell: isWin ? true : undefined,
```

Comment:
Using `shell: true` is required in latest versions of node to run `.cmd` files on Windows.